app/artifact-cas/internal/server/grpc.go

```diff
@@ -93,6 +94,21 @@ func NewGRPCServer(c *conf.Server, authConf *conf.Auth, byteService *service.Byt
 	if c.Grpc.Timeout != nil {
 		opts = append(opts, grpc.Timeout(c.Grpc.Timeout.AsDuration()))
 	}
+	if tlsConf := c.Grpc.GetTlsConfig(); tlsConf != nil {
```

Comment:
nitpick: I tend to add extra lines between blocks for readability. This would mean to add an extra line before ``if tlsConf := ...`

Reply:
The previous blocks are not spaced though. Do you want to only space this block, or all of them?

Reply:
Good point, either way in that case. Thanks